### ISSUE TITLE
chore(linux): Submit new Debian package for our forked ibus

### ISSUE THREAD

**Issue by ermshiperete** [keymanapp/keyman#10106]
See [Slack](https://keymanapp.slack.com/archives/C6Q9WS09G/p1701253696646249?thread_ts=1701251091.616709&cid=C6Q9WS09G) for brainstorming re naming.

**Comment by ermshiperete** [comment]
On hold until we hear back from Debian if they will integrate our patch.

**Comment by ermshiperete** [comment]
An idea to investigate that would allow to work without a patched ibus:

Currently with the patched ibus things work roughly like this: when we get a keypress we send it back to ibus with a special code. Ibus forwards it to the client which sends it back to us.

We have that system service for toggling the capslock indicator which should also be able to function as a "virtual keyboard". I think we could send the keypress to our system service instead of ibus. Since that is a "regular" input source the key press lands at the client which sends it to us without requiring any patches.

**Comment by mcdurdin** [comment]
This sounds like it will be a technical deep-dive but it's a really good idea. Given the friction we have in trying to interact with ibus maintainers, it'd be good to be able to bypass the problem entirely.

**Comment by mcdurdin** [comment]
> See [Slack](https://keymanapp.slack.com/archives/C6Q9WS09G/p1701253696646249?thread_ts=1701251091.616709&cid=C6Q9WS09G) for brainstorming re naming. Suggest to go with kmibus.

Note: Slack messages disappear after 90 days!

**Comment by ermshiperete** [comment]
> > See [Slack](https://keymanapp.slack.com/archives/C6Q9WS09G/p1701253696646249?thread_ts=1701251091.616709&cid=C6Q9WS09G) for brainstorming re naming. Suggest to go with kmibus.
> 
> Note: Slack messages disappear after 90 days!

Good point. So here's the thread from Slack:

**eberhard:** Any suggestions for a name for our forked and patched ibus (package)?
**marc:** Kmbus :grinning:
**marc:** itruck
**marc:** webus
**eberhard:** betterbus
**marc:** biggerbus
**marc:** eyebus
**marc:** ibis
**marc:** byebus
**marc:** Hmmmmmmm
**marc:** arghbus
**marc:** Peter says ecar
**marc:** ibust
**darcy:** whyBus
**darcy:** AIBus
**marc:** not sure if we are very helpful for you @eberhard
**eberhard:** It's good brainstorming :smiley:
**eberhard:** kibus
**eberhard:** keybus
**eberhard:** kaibus

**Comment by ermshiperete** [comment]
Should be done after #10799 and #10634

**Comment by ermshiperete** [comment]
Moving this to a later sprint since I didn't get to this in the assigned sprints. @mcdurdin

**Comment by mcdurdin** [comment]
Hopeful this won't be needed with #10799

**Comment by ermshiperete** [comment]
No longer necessary with #10799 working.
